spec/models/taxonomy/expanded_taxonomy_spec.rb

```diff
@@ -40,8 +46,44 @@ def home_page
   let(:bramley) { fake_taxon("Bramley") }
   let(:cox) { fake_taxon("Cox") }
 
+  describe 'A child and parent taxon not attached to the root taxon' do
+    before :each do
+      @rootless_parent = fake_taxon("Rootless Parent")
+      @rootless_child = fake_taxon("Rootless Child")
+
+      publishing_api_has_item(@rootless_parent)
+      publishing_api_has_item(@rootless_child)
+
+      publishing_api_has_expanded_links(
+        content_id: @rootless_parent["content_id"],
+        expanded_links: {
+          child_taxons: [@rootless_child],
+        },
+      )
+
+      publishing_api_has_expanded_links(
+        content_id: @rootless_child["content_id"],
+        expanded_links: {
+          parent_taxons: [@rootless_parent],
+        },
+      )
+    end
+
+    describe 'Rootless taxonomies' do
+      it 'has one parent, which has no root' do
+        taxonomy = Taxonomy::ExpandedTaxonomy.new(@rootless_child['content_id']).build
+        expect(taxonomy.parent_expansion.children.map(&:internal_name)).to match_array(['i-Rootless Parent'])
+      end
+      it 'has no parents' do
```

Comment:
These tests are quite hard to read, as while the tests are in the same describe block, the object under test is different (so `it` is a different thing in the first test to the second test). Perhaps two describe blocks could be used?

Reply:
changed it to use two blocks with some more description

Reply:
Good stuff :)